java/vector/src/main/java/org/apache/arrow/vector/util/ValueVectorUtility.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.arrow.vector.util;
+
+import org.apache.arrow.util.Preconditions;
+import org.apache.arrow.vector.ValueVector;
+
+/**
+ * Utility methods for {@link ValueVector}.
+ */
+public class ValueVectorUtility {
+
+  /**
+   * Get the toString() representation of vector suitable for debugging.
+   */
+  public static String getToString(ValueVector vector, int start, int end) {
+    Preconditions.checkNotNull(vector);
```

Comment:
Can we also checks the values of start and end?

Reply:
done